### PR TITLE
Residual confounding background

### PR DESCRIPTION
**Pull request by alibow**
Planning to solicit feedback in next Monday's documentation session - please look over in advance if you would like. Thanks :)

### CODE REVIEW

File: docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  :local:
+
+Background and motivation
+-------------------------
+
+As discussed on the :ref:`risk-risk correlation page <risk_correlation>`, risk exposures 
+are frequently correlated at the individual level, with vulnerable populations often 
+experiencing a disproportionate share of risk exposure across several risk factors. 
+Examples include high body mass index and high fasting plasma glucose, tobacco smoking 
+and alcohol use, childhood height and weight, etc.
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst:17]:
```suggestion
and alcohol use, childhood height and weight.
```

Plenty of examples here, no need to have the "etc"

File: docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+Therefore, if we are interested in the effect of a given risk factor (R1) on an outcome, 
+there are often many other risk factors (R2, R3, ... RN) that may be correlated with R1 
+and also have an effect on the outcome (as shown in the figure below). Notably, these 
+"other" risk factors (R2 through RN) meet the definitions of confounding factors in the 
+association between R1 and the outcome (see :ref:`this page <general_dags>` for more
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst:23]:
```suggestion
association between R1 and the outcome (see :ref:`General DAGs <general_dags>` for more 
```

File: docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+  In other words, if we were to stratify a population based on R1 risk exposure, the 
+  difference in the outcome rates between the exposed and unexposed groups would not 
+  equal the difference in outcome rates that would be observed among the exposed 
+  population at baseline and if we were to entirely remove their exposure, making them 
+  unexposed (perhaps through some hypothetical "cure" for the exposure). 
+
+.. todo::
+
+  Demonstrate this mathematically or schematically somehow... any ideas?
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst:51]: You can cook up a dramatic example if R2 is a common cause of R1 and the outcome, e.g. R2 = smoking, outcome = lung cancer, and R1 is something else caused by smoking (I'm having trouble thinking of the right thing now... maybe drinking coffee?)

**Review by aflaxman — APPROVED** [review]

File: docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  exist but have not yet been studied thoroughly enough to meet such a threshold. Such 
+  unknown risks still represent important factors with regard to residual confounding.
+
+The presence of such factors that are associated with R1 and the outcome causes the 
+*observed* association between R1 and the outcome in a population (sometimes referred to 
+as correlation, crude association, or unadjusted association) to be different from the 
+*causal* association between R1 and the outcome (sometimes referred to as the adjusted 
+association). 
+
+  In other words, if we were to stratify a population based on R1 risk exposure, the
```
Comment (zmbc) [inline docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst:43]: Is the indentation here intentional?

File: docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+Whether we are modeling just one or many risk factors in our R1 through RN risk set, 
+the presence of RC can have a significant influence on our models, through the 
+following mechanisms:
+
+1. Bias in estimating the PAF of modeled risk factors on an outcome of interest. The 
+influence of this type of bias is described on the :ref:`population attributable 
+fraction page <pafs>`.
+
+2. Bias in estimating the absolute change in an outcome rate associated with a change
```
Comment (zmbc) [inline docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst:91]:
```suggestion
2. Bias in estimating the absolute change in an outcome rate caused by a change 
```
Maybe?

**Review by zmbc — APPROVED** [review]
Looks great! Just a few small comments.

As mentioned in the real-time review, we may want to revise some of this based on how we end up implementing residual confounding in Vivarium, but this is a very clear explanation of the problem!

**Review by SylLutze — APPROVED** [review]

**Pull request merged**